Makefile

```diff
@@ -51,8 +51,13 @@ install:
 compile: LDF += $(shell GOPATH=${GOPATH} build/flags.sh)
 compile:
 	GOOS=linux GOARCH=amd64 $(GO) build -v -ldflags '$(LDF)'
-	docker build --tag surrealdb/surrealdb:latest .
-	docker push surrealdb/surrealdb:latest
+	docker build --tag surrealdb/surrealdb:latest-amd64 --platform linux/amd64 --build-arg ARCH=amd64 .
+	docker push surrealdb/surrealdb:latest-amd64
+	GOOS=linux GOARCH=arm64 $(GO) build -v -ldflags '$(LDF)'
```

Comment:
Not clear what the Go compiler has to do with building SurrealDB now that it has been rewritten in Rust. Can you explain the purpose of this PR?